byzcoin/bcadmin/cmd_db.go

```diff
@@ -139,6 +143,95 @@ func dbReplay(c *cli.Context) error {
 	return nil
 }
 
+type sumFetcher struct {
+	sum            int
+	bff            byzcoin.BlockFetcherFunc
+	txs            int
+	accepted       int
+	blocks         int
+	timeLastBlock  int64
+	timeLastSum    int64
+	maxTPS         float64
+	maxBlockSize   int
+	totalBlockSize int
+}
+
+func (sf sumFetcher) BlockFetcherFunc(sib skipchain.SkipBlockID) (*skipchain.
+	SkipBlock, error) {
+	return sf.bff(sib)
+}
+
+func (sf sumFetcher) LogNewBlock(sb *skipchain.SkipBlock) {
+	if sf.sum == 1 {
+		log.Infof("Replaying block at index %d", sb.Index)
```

Comment:
Shouldn't you say the following instead?
```
log.Infof("Replayed blocks from index %d to %d", sb.Index - sb.blocks, sb.Index)
```

Reply:
no, if `sf.sum == 1`, there is no summary -> every block is printed.